vermouth/tests/integration_tests/test_integration.py

```diff
@@ -111,6 +111,35 @@ def compare_pdb(filename1, filename2):
 COMPARERS = {'.itp': compare_itp,
              '.pdb': compare_pdb}
 
+def compare_nbparams(fileref, filecomp):
+    """
+    Asserts that two go_nbparams.itp files are functionally identical.
+    """
+    ref = parse_gofiles(fileref)
+    compare = parse_gofiles(filecomp)
+    
+    assert set(ref.keys()) == set(compare.keys())  # assert correct nb pairs
+
+    for key in ref.keys(): 
+        assert(is_equal(list(ref[key]),list(compare[key])))  ##assert correct sigma and eps
```

Comment:
Maybe works:
```suggestion
    assert compare == pytest.approx(ref)
```
Definitely works:
```suggestion
    assert set(ref.keys()) == set(compare.keys())  # assert correct nb pairs

    for key in ref.keys(): 
        assert compare[key] == pytest.approx(ref[key])  # assert correct sigma and eps
```

Reply:
Also, I'd swear I wrote this suggestion before